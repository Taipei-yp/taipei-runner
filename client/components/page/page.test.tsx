import React from "react";
import TestRenderer from "react-test-renderer";
import { Page } from ".";

describe("Component Page", () => {
  const page = TestRenderer.create(<Page className="testclass" />);

  it("correct props", () => {
    expect(page.root.props.className).toBe("testclass");
  });

  it("correct render without childs", () => {
    expect(page.toJSON()).toMatchSnapshot();
  });
});
