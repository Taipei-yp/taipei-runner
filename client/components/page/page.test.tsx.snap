// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Page correct render without childs 1`] = `
<div
  className={
    Object {
      "has": [Function],
      "is": [Function],
      "mix": [Function],
      "split": [Function],
      "state": [Function],
      "toString": [Function],
    }
  }
>
  <div
    className={
      Object {
        "has": [Function],
        "is": [Function],
        "mix": [Function],
        "split": [Function],
        "state": [Function],
        "toString": [Function],
      }
    }
  />
  <div
    className={
      Object {
        "has": [Function],
        "is": [Function],
        "mix": [Function],
        "split": [Function],
        "state": [Function],
        "toString": [Function],
      }
    }
  >
    <div
      className={
        Object {
          "has": [Function],
          "is": [Function],
          "mix": [Function],
          "split": [Function],
          "state": [Function],
          "toString": [Function],
        }
      }
    >
      <div
        className={
          Object {
            "has": [Function],
            "is": [Function],
            "mix": [Function],
            "split": [Function],
            "state": [Function],
            "toString": [Function],
          }
        }
      />
      <div
        className={
          Object {
            "has": [Function],
            "is": [Function],
            "mix": [Function],
            "split": [Function],
            "state": [Function],
            "toString": [Function],
          }
        }
      />
    </div>
  </div>
  <div
    className={
      Object {
        "has": [Function],
        "is": [Function],
        "mix": [Function],
        "split": [Function],
        "state": [Function],
        "toString": [Function],
      }
    }
  >
    <div
      className={
        Object {
          "has": [Function],
          "is": [Function],
          "mix": [Function],
          "split": [Function],
          "state": [Function],
          "toString": [Function],
        }
      }
    />
  </div>
</div>
`;
